packages/api/src/hooks/useOnfido.ts

```diff
@@ -0,0 +1,126 @@
+import { useCallback, useRef, useMemo, useEffect, useState, useId } from 'react';
+import useOnfidoServiceToken from './useOnfidoServiceToken';
+import { ALPHA_2_TO_ALPHA_3, ONFIDO_PHRASES } from '../constants';
+import useSettings from './useSettings';
+import useResidenceList from './useResidenceList';
+import type { SdkHandle, SdkResponse } from '../types/onfido';
+import useOnfidoNotificationEvent from './useOnfidoNotificationEvent';
+import { v4 as uuidv4 } from 'uuid';
+
+/** A custom hook to initialize Onfido SDK  */
+const useOnfido = () => {
+    const [documentsIds, setDocumentIds] = useState<string[]>([]);
+    const [isOnfidoLoaded, setIsOnfidoLoaded] = useState(false);
+
+    const onfidoRef = useRef<SdkHandle | null>(null);
+    const onfidoContainerId = useMemo(() => uuidv4(), []);
```

Comment:
used `uuid` to generate a random id for the onfido container, thus when we want to attach the onfido container to wallets, we just:
```
const { data: { onfidoContainerId } } = useOnfido()
...
return (
   <>
         <div id={onfidoContainerId}></div>
         ...
   </>
)
```

Reply:
https://github.com/binary-com/deriv-app/assets/103016120/07f68fe8-f39d-4066-bd5c-a33f3d71bc63